### ISSUE TITLE
If form has restriced access, and user is not logged in, it should show login form with redirect

### ISSUE THREAD

**Issue by Kaya84** [pluginsGLPI/formcreator#2734]
As written in the subject, it will be better to show login form than no access rights message.

It can simply done by editing front/formdisplay.php and add after line 54
	//If the form has restriced access and user is not logged in, send to login form
	 if (($form->fields['access_rights'] == PluginFormcreatorForm:: ACCESS_RESTRICTED)   && (!isset($_SESSION['glpiID']))){
	  Session::redirectIfNotLoggedIn();
	  exit();
	 }

**Comment by btry** [comment]
Hi

Please do a PR with your suggestion.

**Comment by Kaya84** [comment]
Done.
First time in my github life.
Is it ok?

**Comment by btry** [comment]
Yes, this is OK. I fixed some code style; I will test and merge in a few days.

Thank you for your contribution

**Comment by btry** [comment]
solved with https://github.com/pluginsGLPI/formcreator/commit/fe1eadd68ebcfae57b1d5c5569387d2a7e4e0e51 and https://github.com/pluginsGLPI/formcreator/commit/bdae85a03d32d67ca65baa26e4df83b5f8976e55
